float64 altitude
float64 altitudeRate
geometry_msgs/Pose pose
float64 roll
float64 pitch
float64 yaw